controller/backing_image_manager_controller.go

```diff
@@ -688,7 +688,7 @@ func (c *BackingImageManagerController) isEligibleForPulling(currentBIM *longhor
 			}
 			continue
 		}
-		cksum := util.GetStringChecksum(biName + bimName)
+		cksum := util.GetStringChecksum(biName + bim.Spec.DiskUUID)
```

Comment:
Nice catch! Should we add a comment explain this detail since it is hard to see just from reading the code?

Reply:
Sure. Will add a comment above.

Reply:
Updated